docs/Concepts/Privacy/Onchain-PrivacyGroups.md

```diff
@@ -0,0 +1,47 @@
+description: Onchain Privacy Groups
+<!--- END of page meta data -->
+
+# Onchain Privacy Groups 
+
+Onchain privacy groups use smart contracts to store and maintain the group administration and distribution lists. 
+Onchain privacy groups can have their membership updated. That is, you can add and remove members when 
+using onchain privacy groups. 
+
+!!! important 
+    Onchain privacy groups are an early access feature in v1.4. Do not use in production networks. 
+    
+    The onchain privacy group interfaces may change between v1.4 and v1.5.  
+    
+    Onchain privacy groups are not compatible with [offchain privacy groups](Privacy-Groups.md). You cannot use existing 
+    privacy groups and also create onchain privacy groups in the same chain in v1.4. If you enable
+    onchain privacy groups in a chain with existing privacy groups, you will not longer be able to access
+    the existing privacy groups. 
```

Comment:
> If you enable
    onchain privacy groups in a chain with existing privacy groups, you will not longer be able to access
    the existing privacy groups. 

Onchain groups are always switched on. Onchain groups can be used on the same node as orion groups at the same time. You will have access to all groups at all times.

Reply:
But we'll add the flag soon. I believe we can keep the docs as they are (or update them later).

Reply:
Added the flag in https://github.com/hyperledger/besu-docs/pull/236